.gitignore

```diff
@@ -12,3 +12,4 @@
 /captures
 .externalNativeBuild
 .cxx
+testapp
```

Comment:
```suggestion
testapp
.idea/
```

Reply:
@gipeppo since `.idea` folder got versioned, to remove it from the repository on the next commit you can do a
```
// clear files tracked by git
git rm -r --cached .
// add files to git
git add .
```
and then commit and push.